src/index.ts

```diff
@@ -50,14 +50,22 @@ class VideoSnapshot {
       if (time === 0) {
         video.play();
       } else {
-        // TODO: Handle smart times
         if (typeof time === 'number') {
           video.currentTime = time;
+        } else if (typeof time === 'string') {
+          const duration = video.duration || 0;
+          const smartTimes = {
+            start: 0,
+            middle: duration / 2,
+            end: duration,
+          };
```

Comment:
Do you think we can extract this into own function? That will make it easier to read/test. Something like:

```
getSmartTime(time: CustomVideoTime, duration: number = 0)
```